### PR TITLE
#2438 Ohara Manager should fetch the object data from settings

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2438)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
It seems the used fields (imageName, nodeNames...etc) are move to settings also. So we change the title as well?

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewNodes.js [hunk 1]
```diff
@@ -148,9 +153,11 @@ const OverviewNodes = props => {
...
   handleRedirect: PropTypes.func.isRequired,
   worker: PropTypes.shape({
-    nodeNames: PropTypes.arrayOf(PropTypes.string).isRequired,
-    clientPort: PropTypes.number.isRequired,
-    jmxPort: PropTypes.number.isRequired,
+    settings: PropTypes.shape({
+      nodeNames: PropTypes.arrayOf(PropTypes.string).isRequired,
+      clientPort: PropTypes.number.isRequired,
+      jmxPort: PropTypes.number.isRequired,
+    }),
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesDetailPage/Overview/OverviewNodes.js]:
```suggestion
    }).isRequired,
```

**Comment by konekoya** [comment]
@wu87988622 I will need to update the pipeline part as well. Let me know when you have done your part

**Comment by wu87988622** [comment]
> @wu87988622 I will need to update the pipeline part as well. Let me know when you have done your part

Done, you can pull this PR, thanks

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
> It seems the used fields (imageName, nodeNames...etc) are move to settings also. Should we change the title as well?

Do you mean the PR title @saivirtue if so, feel free to change it :)

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by chia7712** [comment]
@konekoya @wu87988622 I'm going to clean nodes and I will trigger e2e for you.

**Comment by chia7712** [comment]
seems your e2e passes :)

**Pull request merged**